projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -268,6 +319,7 @@ function enqueue_assets() {
 				'post_comment'                    => __( 'Post Comment', 'jetpack' ),
 				'write_comment'                   => __( 'Write a Comment...', 'jetpack' ),
 				'loading_comments'                => __( 'Loading Comments...', 'jetpack' ),
+				/* translators: %1s is applied to the photo-size span class as the full-size image width, and %2s is the height. */
 				'download_original'               => sprintf( __( 'View full size <span class="photo-size">%1$s<span class="photo-size-times">&times;</span>%2$s</span>', 'jetpack' ), '{0}', '{1}' ),
```

Comment:
Here I would recommend placing the translator comment immediately before the translator call:

```suggestion
				'download_original'               => sprintf(
					/* translators: %1s is applied to the photo-size span class as the full-size image width, and %2s is the height. */
					__( 'View full size <span class="photo-size">%1$s<span class="photo-size-times">&times;</span>%2$s</span>', 'jetpack' ),
					'{0}',
					'{1}'
				),
```